go.mod

```diff
@@ -18,20 +18,21 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.14.0
-	github.com/stretchr/testify v1.8.1
-	golang.org/x/exp v0.0.0-20221217163422-3c43f8badb15
-	golang.org/x/term v0.3.0
+	github.com/stretchr/testify v1.8.2
+	golang.org/x/exp v0.0.0-20230129154200-a960b3787bd2
+	golang.org/x/term v0.4.0
 )
 
 require (
+	github.com/OctopusDeploy/go-octodiff v0.0.0-20230329014306-3184489c58de // indirect
```

Comment:
Once I get an approval on Octodiff itself, I'll create a 1.0.0 release/tag for it in github and roll those references through to go-octopusdeploy and the CLI itself. I'll make sure this is done before merging